spec/zoneddatetime.html

```diff
@@ -0,0 +1,1049 @@
+        1. Return _object_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-createtemporalzoneddatetimefrominstance" aoid="CreateTemporalZonedDateTimeFromInstance">
+      <h1>CreateTemporalZonedDateTimeFromInstance ( _zonedDateTime_, _epochNanoseconds_, _timeZone_, _calendar_ )</h1>
+      <p>
+        The abstract operation CreateTemporalZonedDateTimeFromInstance calls the species constructor of the given `Temporal.ZonedDateTime` instance _zonedDateTime_ in order to create a new instance with the given arguments.
+      </p>
+      <emu-alg>
+        1. Assert: Type(_zonedDateTime_) is Object and _zonedDateTime_ has an [[InitializedTemporalZonedDateTime]] internal slot.
+        1. Assert: ! ValidateInstant(_epochNanoseconds_) is *true*.
+        1. Assert: Type(_timeZone_) is Object.
+        1. Assert: Type(_calendar_) is Object.
+        1. Let _constructor_ be ? SpeciesConstructor(_zonedDateTime_, %Temporal.ZonedDateTime%).
+        1. Let _result_ be ? Construct(_constructor_, « _epochNanoseconds_, _timeZone_, _calendar_ »).
+        1. Perform ? RequireInternalSlot(_result_, [[InitializedTemporalZonedDateTime]]).
+        1. Return _result_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-createtemporalzoneddatetimefromstatic" aoid="CreateTemporalZonedDateTimeFromStatic">
+      <h1>CreateTemporalZonedDateTimeFromStatic ( _constructor_, _epochNanoseconds_, _timeZone_, _calendar_ )</h1>
+      <p>
+        The abstract operation CreateTemporalZonedDateTimeFromStatic calls the given _constructor_ with the given arguments to create a `Temporal.ZonedDateTime` instance.
+      </p>
+      <emu-alg>
+        1. Assert: ! ValidateInstant(_epochNanoseconds_) is *true*.
+        1. Assert: Type(_timeZone_) is Object.
+        1. Assert: Type(_calendar_) is Object.
+        1. If ! IsConstructor(_constructor_) is *false*, then
+          1. Throw a *TypeError* exception.
+        1. Let _result_ be ? Construct(_constructor_, « _epochNanoseconds_, _timeZone_, _calendar_ »).
+        1. Perform ? RequireInternalSlot(_result_, [[InitializedTemporalZonedDateTime]]).
+        1. Return _result_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-temporalzoneddatetimetostring" aoid="TemporalZonedDateTimeToString">
+      <h1>TemporalZonedDateTimeToString ( _zonedDateTime_ )</h1>
+      <p>
+        The abstract operation TemporalZonedDateTimeToString returns an ISO 8601 string representation of its argument, including a time zone name annotation and calendar annotation, which are extensions to the ISO 8601 format.
+      </p>
+      <emu-alg>
+        1. Assert: Type(_zonedDateTime_) is Object and _zonedDateTime_ has an [[InitializedTemporalZonedDateTime]] internal slot.
+        1. Let _instant_ be ? CreateTemporalInstant(_zonedDateTime_.[[Nanoseconds]]).
+        1. Return ? TemporalInstantToString(_instant_, _zonedDateTime_.[[TimeZone]], _zonedDateTime_.[[Calendar]]).
```

Comment:
Is this affected by the decision to remove the bracketed time zone from `Instant.prototype.toString()`?  Or is whether to show brackets or not determined by the presence of the calendar argument?

Reply:
Oh right, this is not correct for the case of an offset time zone where the offset needs to be shown again in brackets. So it cannot share TemporalInstantToString after all. (And in #703 Instant is always shown with `Z` anyway)

Reply:
> in #703 Instant is always shown with `Z` anyway

Not if a time zone is supplied to `toString`, where it'd show an offset instead of `Z`.